### ISSUE TITLE
Move one meter in x direction

### ISSUE THREAD

**Issue by krishnarajk** [microsoft/AirSim#4413]
I am trying to move the drone in unreal engine using MoveToPositionAsync function. I want to move the drone 1 meter in the x-direction. What value needs to be given as a parameter in the MoveToPositionAsync function in order to get the expected result. When I give x value as 1, it is moving ~123 uu.
Thanks

**Comment by jonyMarino** [comment]
Hi @krishnarajk! The moveToPositionAsync uses the world frame, not the vehicle one. So, where is your drone located? 
Also, you have to consider that there is a lookahead error. Take a look at #4293

**Comment by krishnarajk** [comment]
@jonyMarino Thanks for the reply. Here is the location of drone in different states.
Before takeoff
![image](https://user-images.githubusercontent.com/11840147/159566978-0288cce5-af39-4ee3-a04b-41096dcaa4cb.png)

After takeoff
![image](https://user-images.githubusercontent.com/11840147/159567018-c80d0493-41f1-4ad0-9e4c-e7d33ef7825f.png)

After moving in x axis, x value given as 1.
![image](https://user-images.githubusercontent.com/11840147/159567133-7afc72db-177b-4b2e-b782-6520404ff079.png)

**Comment by krishnarajk** [comment]
So basically it should move exactly 1 meter? Is it because of the lookahead error?

**Comment by jonyMarino** [comment]
Yes, it is. You can try to decrease it, but it can lead to unwanted behavior in other use cases.

**Comment by krishnarajk** [comment]
How to reduce it?
I tried reducing the value in the "virtual float getDistanceAccuracy()" (from 0.5 to 0.1) function but didn't work for me as expected. 
![image](https://user-images.githubusercontent.com/11840147/159743017-91bfbd5f-5309-4de5-9b42-b659be8d5171.png)

That x value increases to 1.73

**Comment by jonyMarino** [comment]
@krishnarajk, sorry for the delay. It doesn't seem obvious to me to fix the problem by modifying our flight controller. Other alternatives are PX4 or ardupilot. Also, our ROS wrapper has its own PD controller to set the position, and you can adjust its parameters. Do any of these alternatives work for you?
